src/main/markdown/doc/latest/tutorial/style.md

```diff
@@ -32,55 +32,43 @@ When you used webAppCreator to create StockWatcher, it created the application s
 
 1.  Open the GWT module, StockWatcher/src/com/google/gwt/sample/stockwatcher/StockWatcher.gwt.xml.  Notice that the Standard theme is being used by default.
 
-```
-<?xml version="1.0" encoding="UTF-8"?>
-<module rename-to='stockwatcher'>
-  <!-- Inherit the core Web Toolkit stuff.                        -->
-  <inherits name='com.google.gwt.user.User'/>
-
+    ```xml
```

Comment:
Just to be sure: did you test this formatting with pegdown ?
IIRC, it didn't manage very well the triple backticks inside a list (at least with deeper list, maybe it works fine with a one-level list)